source/_components/sensor.thermoworks_smoke.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: "Thermoworks Smoke Sensor"
+description: "Pulls temperature data for a Thermoworks Smoke Thermometer connected with Smoke Gateway."
+date: 2018-08-22 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: thermoworks.png
+ha_category: Sensor
+ha_release: 0.78.0
+ha_iot_class: "Remote Polling"
+---
+
+The `thermoworks_smoke` sensor platform pulls data for your Thermoworks Smoke Thermometer (https://www.thermoworks.com/Smoke).
+This requires a Smoke WiFi Gateway (https://www.thermoworks.com/Smoke-Gateway) with an internet connection.
```

Comment:
Can you make these links (eg `[Smoke WiFi Gateway](https://www.thermoworks.com/Smoke-Gateway)`)